src/schemas/device.py

```diff
@@ -13,6 +13,14 @@ class Config:
         orm_mode = True
 
 
+class DeviceHistorical(BaseModel):
+    max_temp: float | None = Field(None, example="25.1", description="Max Temperature (Celsius)")
+    min_temp: float | None = Field(None, example="25.1", description="Min Temperature (Celsius)")
+    max_humid: float | None = Field(None, example="87.0", description="Max Humidity")
+    min_humid: float | None = Field(None, example="87.0", description="Min Humidity")
```

Comment:
Delete the `None`. When storing device historical, `None` should ever be entered.
```py
max_temp: float = Field(example="25.1", description="Max Temperature (Celsius)")
```

Reply:
Updated!